kpet/targeted.py

```diff
@@ -86,9 +86,9 @@ def get_test_cases(src_files, dbdir):
     test_cases = set()
```

Comment:
A comment on commit message: could you please use `tag:` format instead of `[tag]` format for the subject?

Reply:
Sure thing, I hope remember it :grimacing: